### PR TITLE
Aarnq/psp fixes

### PR DESCRIPTION
**Pull request by aarnq**
**What this PR does / why we need it**:

Fixes for the PSP changes.

**Checklist:**

- [x] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [x] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [ ] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.
- Chart checklist (pick exactly one):
    - [ ] I upgraded no Chart.
    - [ ] I upgraded a Chart and determined that no migration steps are needed.
    - [ ] I upgraded a Chart and added [migration steps](https://github.com/elastisys/compliantkubernetes-apps/blob/main/migration).

**Pipeline config** *(if applicable)*
If you change some config options (e.g. add/rename variable or change the default value) you may need to update the config used by the pipeline in `pipeline/config`.

### CODE REVIEW

File: helmfile/values/gatekeeper/gatekeeper.yaml.gotmpl [hunk 1]
```diff
@@ -3,6 +3,69 @@ validatingWebhookTimeoutSeconds: 5
...
+      - deployments
+      - deployments/scale
+      - replicasets
+      - replicasets/scale
+      - replicationcontrollers
+      - replicationcontrollers/scale
+      - statefulsets
+      - statefulsets/scale
+      - services/proxy
+      - nodes/proxy
```
Comment (robinAwallace) [inline helmfile/values/gatekeeper/gatekeeper.yaml.gotmpl:67]: Could you also add for my hnc mutation?
```suggestion
      - nodes/proxy
      - hierarchyconfiguration
```

**Review by robinAwallace — APPROVED** [review]

**Review by OlleLarsson — APPROVED** [review]

File: helmfile/values/podsecuritypolicies/common/fluentd.yaml.gotmpl [hunk 2]
```diff
@@ -6,9 +6,7 @@ constraints:
         app.kubernetes.io/instance: fluentd-forwarder
       allow:
         allowPrivilegeEscalation: true
-        allowedCapabilities:
-          - AUDIT_READ
-          - SYSLOG
+        allowedCapabilities: []
```
Comment (robinAwallace) [inline helmfile/values/podsecuritypolicies/common/fluentd.yaml.gotmpl:9]: Are these not needed anymore?

Comment (OlleLarsson) [reply helmfile/values/podsecuritypolicies/common/fluentd.yaml.gotmpl:9]: They aren't set in the securityContext so its not possible to validate, thus also why the capabilities mutation has been disabled.

Comment (robinAwallace) [reply helmfile/values/podsecuritypolicies/common/fluentd.yaml.gotmpl:9]: Ah okay

**Review by Xartos — APPROVED** [review]
LGTM

**Review by OlleLarsson — APPROVED** [review]

**Pull request merged**